src/test/ReactTestUtils.js

```diff
@@ -452,6 +455,12 @@ function makeSimulator(eventType) {
       EventPluginHub.enqueueEvents(event);
       EventPluginHub.processEventQueue(true);
     });
+    // Normally extractEvent enqueues a state restore, but we'll just always do
+    // that.
```

Comment:
This bypasses the ChangeEventPlugin and creates its own synthetic event. So it doesn't go through that logic.

How would you suggest that I leave it in ChangeEvent? Expose something only for testing?